FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1033,6 +1036,355 @@ paths:
       security:
       - OAuth2: [fraihmwork.component.write, fraihmwork.component.admin]
 
+  /monitor/v1/relationship/definition:
+    get:
+      tags:
+      - Relationships
+      - Monitor
+      summary: Retrieve all of the relationship definitions active in FRAIHMWORK
+      operationId: findAllRelationshipDefinitions
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RelationshipDefinition'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden - client is not authorized to read components
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      deprecated: false
+      security:
+      - OAuth2: [fraihmwork.component.read, fraihmwork.component.write, fraihmwork.component.admin]
+    post:
+      tags:
+      - Relationships
+      - Monitor
+      summary: Add a new relationship definition
+      description: Relationship definitions are usually defined in system configuration, but there are scenarios where a user or integrator will want to add new relationship definitions for a specific deployment. This endpoint facilitates that behavior.
+      operationId: postRelationshipDefinition
+      requestBody:
+        description: JSON description of the new relationship definition
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RelationshipDefinition'
+      responses:
+        201:
+          description: Successfully posted new component information
```

Comment:
```suggestion
          description: Successfully posted new component definition
```

Reply:
**component relationship definition**, instead?

Reply:
Everyone is wrong! But my new change should be right!